README.md

```diff
@@ -1,11 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet) 
 
-# Bookstore: create and remove books.
+# Bookstore: React and Redux Project.
 
-For this repository, I've created a project to create a React project using Redux, the main objective is to learn the way to create an application to store, use actions and reducers in React, to understand the concepts about state management, and connect React and Redux.
+For this repository, I've created a project to create a React project using Redux, the main objective is to learn the way to create an application to store, use actions and reducers in React, to understand the concepts about state management, connect React and Redux, connect an existing API via Redux and use the data to display something in the UI, and apply React best practices and language style guides in code.
```

Comment:
- [ ] Please deploy the project online and include the live deploy link in the readme.